src/resources/schema/definitions.yml

```
@@ -2954,7 +2954,7 @@
                     #     string:
                     #       pattern: '^U\+[0-9A-F]{4}(?:-[0-9A-F]{4})?(?:,U\+[0-9A-F]{4}(?:-[0-9A-F]{4})?)*$'
         description: >
-          The font files to include. These can be local or online.
+          The font files to include as an array (*i.e.*, `- path: <path>`). These can be local or online.
```

I'm not usually fussy about Latin abbreviations, but I think this "i.e." ("in other words") should be "e.g." ("for example") because listing the font files without the `path:` prefix should also work.

---

I did not try without `path` as it's not even shown anywhere in brand.yml documentation thus this "wording".
https://posit-dev.github.io/brand-yml/brand/typography.html#comprehensive-example-with-font-definitions

---

```suggestion
          The font files to include as an array (`- path: <path>` or `- <path>`). These can be local or online.
```

No latin abbreviation and both cases.

---

Nice!

---

I'll make the change and build the files then.

---

43f93e13c7c21e06f318c527d31538f909a086c7